Assets/MixedRealityToolkit.Tests/EditModeTests/Core/Extensions/CameraExtensionTests.cs

```diff
@@ -62,14 +111,38 @@ public void TearDown()
             GameObjectExtensions.DestroyGameObject(testCamera.gameObject);
         }
 
+
+
+        [Test]
+        public void TestIsInFOVConeCached()
+        {
+            for (int i = 0; i < TestColliders.Count; i++)
+            {
+                var test = TestColliders[i];
+                Assert.AreEqual(test.ShouldBeInFOVCamera1, testCamera.IsInFOVConeCached(test.Collider), $"TestCollider[{i}] did not match");
+            }
+        }
+
+        [Test]
+        public void TestIsInFOVConeCachedSecondCamera()
+        {
+            for (int i = 0; i < TestColliders.Count; i++)
+            {
+                var test = TestColliders[i];
+                Assert.AreEqual(test.ShouldBeInFOVCamera1, testCamera.IsInFOVConeCached(test.Collider), $"TestCollider[{i}] did not match");
```

Comment:
Why is the testCamera done twice but not the second camera?

Reply:
Good catch, I was testing that the cache was getting hit. There should be only one call to testCamera.IsInFOVConeCached